layers/meta-balena-beaglebone/recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -0,0 +1,10 @@
+#SRC_URI = "https://github.com/beagleboard/ti-linux-firmware.git;branch=09.00.00.007"
```

Comment:
Why was a different version of linux-firmware needed?

Reply:
This was part of the debugging of the `wlcore` issue (which turned out to be unrelated to the `linux-firmware`). We can probably remove this and retry.